crime-commons-mod-schemas/build.gradle

```diff
@@ -13,14 +13,9 @@ semver {
 def versions = [
         jacksonVersion      : "2.17.0",
         validationapiVersion: "3.1.0-M2",
-        crimeCommonsClasses : "3.18.0"
-
-
+        crimeCommonsClasses : "3.28.0"
```

Comment:
In your comments you mentioned that the version has been set to the latest version to prevent having to publish both modules? Thought I'm not sure I quite understand, doesnt the enum changes still have to be published to be available to these changes here?

Reply:
The plan was to use the latest `crime-commons-classes` version when building `crime-mod-schemas`. However, I ran into some issues and decided to spin up another ticket instead. Therefore, I just needed to update the dependency version in `crime-mod-schemas`. I'll update the PR description to reflect this.

Reply:
Ok thats fine.